_drafts/restless-restfull.md

```diff
@@ -0,0 +1,454 @@
+# The RESTless struggle for RESTfullness
+
+This post will be about my history trying to understand
+REST and RESTfullness (whatever that is, spoiler alert, I'm
+not going to define what it is), what worked for me and
+what sidetracked me.
+
+As I evolve my ideas over time do not confuse that with
+me being sure of anything, I'm still not sure of shit
+until now, the only thing I'm sure is that going back
```

Comment:
what is 'going back' here?

Reply:
to the old ideas that I had before =P, will try to make it more clear.